backend/sirius-components-charts/src/main/java/org/eclipse/sirius/components/charts/barchart/elements/BarChartElementProps.java

```diff
@@ -107,13 +116,19 @@ public Builder keys(List<String> keys) {
             return this;
         }
 
+        public Builder style(BarChartStyle style) {
+            this.style = Objects.requireNonNull(style);
+            return this;
+        }
+
         public BarChartElementProps build() {
             BarChartElementProps barChartElementProps = new BarChartElementProps();
             barChartElementProps.id = Objects.requireNonNull(this.id);
             barChartElementProps.descriptionId = Objects.requireNonNull(this.descriptionId);
             barChartElementProps.label = Objects.requireNonNull(this.label);
             barChartElementProps.values = Objects.requireNonNull(this.values);
             barChartElementProps.keys = Objects.requireNonNull(this.keys);
+            barChartElementProps.style = this.style;
```

Comment:
// Optional on purpose
(to be consistent with all other widgets)

Reply:
ok